### DISCUSSION TITLE
S100105 - Coleman Saluspa Havana compatibility report - OK

### DISCUSSION THREAD

**Discussion by choumarin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#293] [General]
Just wanted to report I've installed this in my spa and everything seems to be working perfectly.
Used the v1 PCB with the TXS0108E, didn't add any resistors. The code is using `#define MIAMI2021  //no hydrojets` .
The only thing that seems a bit broken is the sound, it's a very faint and very undefined chip. But I can live with that.

Thanks for this super project, I can now integrate with home assistant to only power the spa during non peak hours.

**Comment by visualapproach** [comment]
Thank you @choumarin ! Sorry for not commenting before.
